tools/verify/e5-t18d-desktop-recovery.mjs

```
@@ -0,0 +1,238 @@
+  for (const result of outcomes) if (result.status === "rejected") throw result.reason;
+  cases.sort((left, right) => left.label.localeCompare(right.label));
+  assert.equal(execFileSync("git", ["rev-parse", "HEAD"], { encoding: "utf8" }).trim(), head, "HEAD moved during recording");
+  for (const [file, digest] of Object.entries(sources)) assert.equal(await hashFile(file), digest, `${file} moved during recording`);
+  await writeFile(path.join(out, "desktop-recovery.json"), JSON.stringify({ schema: "wasm-vm.e5-t18d.recovery.v1", task: "E5-T18d", head,
+    predictions: ["new PID and fresh readiness after each crash", "exactly three crashes end at visible tty1 getty", "broken config never claims readiness"],
+    sources, publication, browser: browser.version(), cases, passed: true }, null, 2) + "\n");
+  console.log("E5T18D_PASS=2");
+} finally {
+  clearInterval(progressTimer);
+  await browser?.close();
+  if (server) { try { process.kill(-server.pid, "SIGTERM"); } catch (failure) { if (failure.code !== "ESRCH") throw failure; } }
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)